### ISSUE TITLE
Make README less verbose

### ISSUE THREAD

**Issue by rye** [kotct/dot#79]
The README file is a little bit verbose.  Ideally, it should just include some basic stuff.  Principles could be extracted to the Wiki or to another .md file.

Open to feedback and ideas.

**Comment by cg505** [comment]
Yeah, lots of stuff should be extracted to the wiki or CONTRIBUTING.md.
